functions/icons.zsh

```diff
@@ -92,8 +93,9 @@ case $POWERLEVEL9K_MODE in
       TEST_ICON                      $'\uF291'              # 
       TODO_ICON                      $'\u2611'              # ☑
       BATTERY_ICON                   $'\U1F50B'             # 🔋
-      OK_ICON                        $'\uF23A'              # 
-      FAIL_ICON                      $'\uF281'              # 
+      HDD_ICON                       $'\uF0A0 '             # 
+      OK_ICON                        $'\uF00C '             # 
```

Comment:
I would prefer not to change codepoints for unrelated Icons. If there are better icons, they should be updated in a separate pull request.